### DISCUSSION TITLE
Vocab - HES Specialty

### DISCUSSION THREAD

**Discussion by lizzieLepoo** [OHDSI/Vocabulary-v5.0#811] [Q&A]
I'm pretty new to all-things-OHDSI - but being an NHS (National Health Service) analyst in the UK and trying to think about applying CDM standards to some data, my eye was drawn to the "HES Specialty" vocab.

I was wondering whether there might be any information on how the concepts in this vocab have been flagged - ie standard/non-standard, which class, which domain.

The way I am looking at the download from Athena doesn't easily fit in my existing mental map of how specialties are handled within NHS data, so I was hoping to get a better understanding of how these have been assigned in the Athena arena. 

Thanks

**Comment by cgreich** [comment]
@lizzieLepoo:

There is not much magic here. The HES specialty codes are either standard (used in the data), non-standard but mapped to something standard (e.g. 71 "Addition Service" is mapped to Medicare Provider code 79 "Addiction Medicine") or it really isn't a specialty but a Visit (e.g. 520 Postnatal clinic) or it is some coding oddity dumped into Observation (e.g. 190 "Not a treatment function"). The vast majority are standard specialties.

Where do you see the problem?

**Reply by lizzieLepoo** [reply]
Thanks for your reply @cgreich 

I have a few worked examples which may (or may not!) give a better idea of my overall curiosity - and really this was prompted by latching onto the fact that Athena contained a Vocab that was in many ways familiar to me, so forgive anything that seems strange.

1. Ophthalmology, in NHS data, would be recognised as coded value 130
The "HES Specialty" vocab entry for this, in the Physician Specialty class, is set as Non Standard
If I look within CONCEPT_RELATIONSHIPS I can, however, reach the standard concept for Ophthalmology, which is in the "Medicare Specialty" vocab

2. Similarly, we have Haematology (303) and Clinical Haematology (823) in NHS data
The "HES Specialty" vocab entry for these, in the Physician Specialty class, are both set as Non Standard
If I look within CONCEPT_RELATIONSHIPS I can, however, reach the standard concept Hematology, which is in the "Medicare Specialty" vocab, for both of these

3. We have Paediatric Dentistry (142) in NHS data
The "HES Specialty" vocab entry for this, in the Physician Specialty class, is as Non Standard
If I look within CONCEPT_RELATIONSHIPS I can, however, reach the standard concept Pediatric Dentistry, which is in the "NUCC" vocab (not Medicare)

4. However, Paediatric Ophthalmology is code 216 in NHS data
The "HES Specialty" vocab entry for this, in the Physician Specialty class, is set Standard
If I look within CONCEPT_RELATIONSHIPS, this concept only maps to itself

So I guess my questions (bearing in mind I am really new to this all!) are - 

1) is "HES Specialty" from the UK coding standards (for us, HES = "Hospital Episode Statistics" - a data service provided by NHS England)

2) is the rule of thumb for something being a "Standard Concept" that it was the first version of that concept that was created? Then all other 'same' concepts will be non-standard, but will map to that originating one - meaning that any time a definitively new concept occurs, it becomes the "Standard Concept" itself - regardless of which vocab it may belong to.

And finally a linked question
3) in 2021 HES made some changes and additions to the codes / descriptors for some specialties. Is this something that I (or someone) should be letting Athena know about?

**Reply by cgreich** [reply]
1. We don't make entire vocabularies standard. Standard is an attribute of a concept. In this case of specialties, we use a hodgepodge of vocabularies and cooked a stew. Whenever there was duplication, like in your cases, we picked one and mapped the other ones to it. Your cases 1-3 is where the Medicare Specialty or NUCC ones were picked to be "more equal than the others". Case 4 is where HES won. But it is arbitrary and has no consequence. 
2. No. Among equivalent concepts from different vocabularies, one becomes the standard and the others are mapped to it.  For example, HES "Dental medicine" is mapped to Medicare Specialty "Dentistry" (as is NUCC "Dental Providers"). Which one is the boss of the others is pretty arbitrary (usually the one with the best name won), but once established you **have to** follow the mapping and use the standard in your data. Hopefully, we really picked equivalents to do this mapping.
3. Yes, please. Would you mind?

**Reply by lizzieLepoo** [reply]
Sorry it's taken so long to reply to point 3. I attach a list of all the ones on the NHS Data Dictionary that don't appear in the Athena "HES Specialty" vocab - I've flagged them with two flags - as we have a clinician MAIN SPECIALTY and, where there are well established more granular specialties of treatment within that, we have TREATMENT FUNCTION CODE - when I checked the existing entries in the Athena vocab I found some which were both, and some which were only one, or only the other, so I am including all of them in case useful.
[Athena HES Specialty check.xlsx](https://github.com/OHDSI/Vocabulary-v5.0/files/11444716/Athena.HES.Specialty.check.xlsx)
